src/category_theory/adjunction/over.lean

```diff
@@ -0,0 +1,51 @@
+/-
+Copyright (c) 2021 Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bhavik Mehta
+-/
+import category_theory.over
+import category_theory.limits.preserves.basic
+import category_theory.limits.creates
+import category_theory.limits.shapes.binary_products
+import category_theory.monad.products
+
+/-!
+# Adjunctions related to the over category
+
+Construct the left adjoint `star X` to `over.forget X : C / X ⥤ C`.
```

Comment:
Should we introduce some notation for `C / X`? If not, maybe it's better to just write `over X`.

Reply:
Amended, thanks.